scripts/generator/templates/template-release-notes.md

```diff
@@ -81,7 +81,7 @@ Vaadin also provides [commercial support and warranty](https://vaadin.com/soluti
 - Chrome (evergreen)
 - Firefox (evergreen)
    - Firefox Extended Support Release (ESR)
-- Safari 15 or newer
+- Safari 15 (latest minor version in each major series) or newer.
```

Comment:
I realize this could be even clearer if the parenthesis would be at the end, i.e. `Safari 15 or newer (latest minor version in each major series).` since the latest minor rule also applies for the "or newer" part.

Reply:
done